.github/labeler.yml

```diff
@@ -0,0 +1,34 @@
+# Config file for pr-auto-label workflow
+
+# enable auto-labeler on issues, prs, or both.
+enable:
+  prs: true
+
+# Labels is an object where:
+# - keys are labels
+# - values are objects of { include: [ pattern ], exclude: [ pattern ] }
+#    - pattern must be a valid regex, and is applied globally to
+#      title + description of issues and/or prs (see enabled config above)
+#    - 'include' patterns will associate a label if any of these patterns match
+#    - 'exclude' patterns will ignore this label if any of these patterns match
+labels:
+  'kind/BUGFIX':
+    include:
+      - '(^|[^\s])/kind (fix|bugfix)'
+    exclude: []
+  'kind/ENHANCEMENT':
+    include:
+      - '(^|[^\s])/kind enhancement'
+    exclude: []
+  'kind/FEATURE':
+    include:
+      - '(^|[^\s])/kind (feat|feature)'
+    exclude: []
+  'kind/DOCUMENTATION':
+    include:
+      - '(^|[^\s])/kind (doc|docs|documentation)'
+    exclude: []
```

Comment:
Let's remove `kind/DOCUMENTATION` because we don't add such information to our changelog.

At the same time, we're missing `kind/CHANGE`